client/source/js/modules/common/after-render-directive.js

```diff
@@ -0,0 +1,17 @@
+define(['angular'], function (angular) {
+  'use strict';
+
+  return angular.module('app.common.after-render', [])
+	.directive('afterRender', [ '$timeout', function($timeout) {
```

Comment:
Tabs are off in this file, and please remove inline annotation syntax - ngAnnotate takes care about it before minifying.